### DISCUSSION TITLE
Editing the handbook

### DISCUSSION THREAD

**Discussion by kubark42** [iltis42/XCVario#244] [General]
Is there any appetite for modifying the English handbook? As I read through, I notice it's an excellent piece of documentation, by far one of the best I've ever come across in an open-source project. That being said, I notice there are some areas where I was confused, sometimes by the way the sentence was phrased, sometimes by a lack of imagery, and sometimes by words which look like they might have been translated into English too literally from their original language (TIL that "crank" is the German word for thermalling). 

Based on some passages I fed into google translate, I think-- and correct me if I'm wrong-- that the English version is a machine-translated document. If so, that poses certain difficulties to adjust the English copy, as any modifications or refinements will either 1) get clobbered when a new version comes out; or 2) splinter the manual maintenance effort as now it's much more complicated than getting google to do it (Source: a lot of friends in the EU Parliament translation dept).

The English is close enough in about 95% of cases. I only had to ask a German glider pilot to explain one thing to me (The sentence "The procedure can only be used if there is at least a crank share of 20%." in the `Auto-Deviation` section). And of course, the machine translation will only get better over time. So this might be unresolvable, at which point a link in the handbook to a Github page documenting the machine translation oddities might be sufficient.

**Comment by iltis42** [comment]
Documentation will be on wordpress soon, maybe some authors can join, out native lang is not english, hence we want to make this the source for automatic translations in other languages. If english was perfect, guess automatic translations in other languages is acceptable:
https://xcvario.com/docs-category/vario/
